### PR TITLE
Update base images and removed wrong hbase jmx config pattern

### PR DESCRIPTION
**Pull request by maltesander**
- Updated all base images (java-base, ubi8)
- Collect all HBase JMX metrics for now, because the Atruvia testing demonstrated that the old filters didn't match anything

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -207,21 +207,25 @@
             {
                 'product': '377',
                 'java': '11',
+                'sha256': '7929833412c331fc23cde0e23ca730d652c0be61a8a69c8a82b2af937a3fbd4e',
```
Comment (teozkr) [inline conf.py:210]: Can we rename this to `java_base_image_sha256` or something like that?

File: hbase/stackable/jmx/region-server.yaml [hunk 2]
```diff
@@ -1,7 +1,2 @@
 rules:
-  - pattern: HadoopNamespace_([^\W_]+)_table_([^\W_]+)_region_([^\W_]+)_metric_(\w+)
-    name: HBase_metric_$4
-    labels:
-      namespace: "$1"
-      table: "$2"
-      region: "$3"
+  - pattern: ".*"
```
[the remark this answers is not in the document]
Comment (maltesander) [reply hbase/stackable/jmx/region-server.yaml:2]: The pattern was wrong and basically filtered out all metrics from hbase (came up with atruvia demo). So for now better just forward everything until we come up with a better / tested config.

File: conf.py (hunk 1 above)
Comment (maltesander) [reply conf.py]: done https://github.com/stackabletech/docker-images/pull/250/commits/ae7225f1aa0764d15aaf4ab985faabb07593ff2a

File: hbase/stackable/jmx/region-server.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (teozkr) [reply hbase/stackable/jmx/region-server.yaml:2]: That's fine to me, I'd just like to see that stated in at least the PR description (future commit message).

**Review by teozkr — APPROVED** [review]

**Pull request merged**